extensions/amp-facebook/0.1/amp-facebook.js

```diff
@@ -51,8 +53,15 @@ class AmpFacebook extends AMP.BaseElement {
   layoutCallback() {
     const iframe = getIframe(this.win, this.element, 'facebook');
     this.applyFillContent(iframe);
+    const sentinel = generateSentinel(window);
+    iframe.name = encodeURI(JSON.stringify({
```

Comment:
Can't use such a predictable name, or we'll run into [iframe caching](https://github.com/ampproject/amphtml/pull/2955).